### PR TITLE
U/stalderriel/ptc

### PR DESCRIPTION
**Pull request by bstalder**
much better commenting, and it works!

### CODE REVIEW

File: eotestanalysis_example.ipynb [hunk 1]
```diff
@@ -0,0 +1,597 @@
...
...IyKghUhFp8gImzwLuCwQ5PlZERp1XRN6AW2ED4Beh3Vd5318UkWmBPPOBDwN9wNX+dnU12c/zneD5RORc4HU4S+Mfiy9p7Qn68JgFxGgE/LocrtdZxeRgJInVvWxSjckYbwa+KSJ/AlYDW3Ezb08FDgA2Au8LpP8ecLCI3IdbzgzcrFs/pt2XVfW+4AlU9T4R+R5utYuVInIjbgm0C4DpwEdDq2L45zkHOA94QETuxMXWOx838/fv07QqBox+0zKnWUcwPIBRX5hi47A6bM+zpDCZG9VQ9O4ADsLFzDsGF7KkB+f/9nPgh6q6LZD+58DbgONxQ6jjgFdwq2hcqar3RJ1EVT8pIo/jLHjvB4Zw6+t+V1VvjUjfJyJnAJ8DLsJZCztxYV0uU9WnKix31QlOxrDG6QjKwuRSn9h9c2RdBlkvf1LYS4ZRsaKnqk8AHykh/U+Bn5Z5rqXA0hLS78KFcgmHc0kVYUXGfChy2LBf/RMO2ppVTOE1kiAczcHqYPaIYzKGMQZBK55NQBiJKb31jynqOaw+G3ET5RJkZItqr4xhlEk+S55ZAQyjvjHXAyMpLEi3AWbRSw353rKy3EBt5nH9Ew4rkkXCfrdZlYMRP+YOZIApekYdYEMP9Us48HeW751NtDKSwNqgYUO3KcMcZ3Pkk0GWZVKP2P1ymGXFSIpgG7T2mD1M0UsJFlolR9bL32hYnc5hcjDiJuinZ20xm9jQbQoIh5/I+lu/DTE0HnZPs112IzmCyp0pednEFL0UkG+2baH9jU4+vxJbJ7S+CIdVyWpHY+FljCTxn5v27MwmpuilhHAIBuscHeE3Uesw64tg55LlTibLZTfSg82Czyam6KWEqIkHWW6Ipsw1BkHFPKv31F5OjDRgk9qyiyl6KcIaXo6g83CWFd56xyx6DqvHRtJY/csupuilmKwqfuHAslmVQ6ORVWu11WPDMJLEFL2UEJxtm6VOMB9ZH+5rNLI+bGTWaSNJgu4DVgezhyl6KSHY+LLWCYaxB5HRiJivnpEkthRfdjFFL2VkPayKjy171jhkdcg2iNVnI0mC0RzMlSB7mKKXMsJhVrKOPZQagyyv+hJV5izKwUgWe7nILqbopQxrjCOxlULqn6jYkFnC6q+RNBaXNduYopcygpMQrIMwjMbB2rORFMFwVUb2aEn6Aoz8ZPnNK/wGakO49Y1NNsp
```
Comment (SimonKrughoff) [inline eotestanalysis_example.ipynb]: get a the --> get at the

Comment (SimonKrughoff) [inline eotestanalysis_example.ipynb]: This looks pretty good until ~150000 and then the variance shoots up in the last bin.  Is that saturation, or something else?

**Review by SimonKrughoff — APPROVED** [review]
Looks great.  This does violate the "clear all outputs" guidance, but it is so image driven it seems ok to me in this case.

**Comment by bechtol** [comment]
This looks to be an excellent introduction to ISR utilities.

At cell `amp=raw0.getDetector()[8]`, it seems that `raw0` has not been defined. I'm getting an error when running through the notebook. Maybe `raw0` -> `bias`? Although it seems that the main point of this line is to get the amp geometry, so it seems that any valid raw image would work for this purpose.

**Comment by SimonKrughoff** [comment]
Just note that there are [ISR utility functions](https://github.com/lsst/ip_isr/blob/master/python/lsst/ip/isr/isrFunctions.py) and [an `IsrTask`](https://github.com/lsst/ip_isr/blob/master/python/lsst/ip/isr/isrTask.py) in `ip_isr`.

**Pull request merged**